examples/js/loaders/GLTFLoader.js

```diff
@@ -14,6 +14,7 @@ THREE.GLTFLoader = ( function () {
 
 		this.dracoLoader = null;
 		this.ddsLoader = null;
+		this.geometryUtils = null;
```

Comment:
Hmm, what do you think about waiting to see if anyone complains before adding this?

Reply:
I'm fine with that 👍 . 

How about logging a warning instead that triangle strip and fan are not supported by the loader anymore? Otherwise the loader would silently ignore these draw modes.

Reply:
I've force pushed a new version of the PR with the suggested changes.

Reply:
I have given a little bit more thought to this and, considering that it's not that much code, I think we should duplicate the `BufferGeometryUtils.toTrianglesDrawMode()` code inside `GLTFLoader` so it just works.

Reply:
Okay, I've embedded a slightly smaller version of `toTrianglesDrawMode()` into the loader. Since we know how `toTrianglesDrawMode()` is called in the loader, some error checking is obsolete.